### PR TITLE
Properly installing `chromedriver` on Mac

### PR DESCRIPTION
**Pull request by s1l3ntcat**
On newer versions of Mac, chromedriver can be installed using `brew cask install chromedriver`

### CODE REVIEW

**Comment by Bisaloo** [comment]
Thank you for your contribution. You seem to have inadvertently removed the translations submodule. Could you revert this change please?

**Comment by s1l3ntcat** [comment]
So, I was going to restore the translations submodule by finding the SHA1 hash of the commit which I can find with `git log`. However, this is also that shows up when I look up my name:

    commit a9c8ce43f645aee67ed94ef094d3850d5981f766 (HEAD -> master, origin/master, origin/HEAD)
    Author: A Silent Cat <silentcat@protonmail.com>
    Date:   Sat Jul 14 13:41:37 2018 -0500

    Removed seemingly redundant colors in script.py

I also notice that on the top of this page it says "s1l3ntcat wants to merge 2 commits into EFForg:master from unknown repository." 

TLDR; I can't find this commit in the commit history, and therefore I can't undo my deletion of the translation module.

**Comment by s1l3ntcat** [comment]
As a side note, no, I didn't delete my fork: https://github.com/s1l3ntcat/https-everywhere/tree/master/test

**Comment by s1l3ntcat** [comment]
Most likely, I'm just going to close this pull request and reopen another one.
